### ISSUE TITLE
https://kuramanime.com/?

### ISSUE THREAD

**Issue by ModooMarble** [Sainan/Universal-Bypass#1617]
Before you report a website that isn't bypassed yet, please use the search function to ensure that the site wasn't already reported: https://github.com/Sainan/Universal-Bypass/issues?utf8=%E2%9C%93&q=ENTER+DOMAIN+HERE
link: https://kuramanime.com/?b935a369fa=ODROOEIzZ2F5SW9KZ1Rlck5jNGFRK3lTM3NoUFVpWC8xZUd1cnlNaEVLNWF0cE5KblkwTlpxWnRDNzJMRlRScmlQRUpQdVp4MVQrRURWS3cvb0VJT0Y5NXdOMSs3TFRJRklGWS9CSHhrQXByVGdHZHh6OEh0RS9OYjdGRzV5cTBzV1hBcXVsSXdKU2NlUWJVZFVPMFRRPT0=

from:https://kuramanime.com/darwins-game/#batch

If that is not the case, please **provide an example link.** We need to be able to see what there is to be bypassed, otherwise there can't be a bypass.

**Comment by respecting** [comment]
Possibly #1607, same scheme.

**Comment by Sainan** [comment]
Yes, it just needs a bypass clipboard to fix the path. Really can't be bothered, tho.
